acceptancetests/utility.py

```diff
@@ -317,6 +317,10 @@ def add_basic_testing_arguments(parser, using_jes=False, deadline=True,
     parser.add_argument('--keep-env', action='store_true',
                         help='Keep the Juju environment after the test'
                         ' completes.')
+    parser.add_argument('--existing', action='store_true',
```

Comment:
So when using --existing it just uses whatever controller is currently selected?

I suggest enhancing this slightly and make the controller name an arg passed (perhaps with a default so it falls back to using just whatever exists).
Naming the controller means that one can run many concurrent tests on the same machine, using the same juju bin and JUJU_DATA on different controllers (i.e. different clouds etc.).

If we rely on using the selected controller that would require a unique JUJU_DATA per controller etc.

It would be really cool to be able to do:

JUJU_DATA=/blah ./assess_model_migration --existing my-aws-controller ... && JUJU_DATA=/blah ./assess_model_migration --existing some-maas-controller

Thoughts?